### DISCUSSION TITLE
Unable to load json files when executing.

### DISCUSSION THREAD

**Discussion by quentinalt-dintec** [c-jimenez/open-ocpp#209] [Q&A]
We have followed all the instructions to build into our project and it is all compiling well.
But when we run the executable, we have errors telling us that it cannot find the json files.

Any help about that issue ?

**Comment by c-jimenez** [comment]
Hi,
The location of the JSON file is specified in the configuration file of each of the examples.
If you run the examples from within the bin/gcc_native folder the example should run without modifying the default configuration.

**Comment by quentinalt-dintec** [comment]
Look what I am getting :

![image](https://github.com/c-jimenez/open-ocpp/assets/172390743/4ffbba26-f761-4fc9-85d0-c74bbbadd8f6)

and this for like 80 json files

Dou you what could be the reason for this ?

**Comment by quentinalt-dintec** [comment]
We are actually struggling to compile the examples...
We have followed every steps to get this work but we are kind of stuck.
Could you help us ?

**Comment by c-jimenez** [comment]
From the screenshot, it seems that you are using the `CentralSystem `object. 
To instantiate it, you must have provided an implementation of the `ICentralSystemConfig `interface.
This interface defines the `jsonSchemasPath() `method which should return the directory which contains the json schemas.
If you have installed the library on you system using the `make install-` command, the directory should be : `/usr/local/include/openocpp/schemas/`
If you have not installed the library, you can point to the `schemas` directory of you source tree.

**Comment by c-jimenez** [comment]
What are your issue when you are compiling the examples?
Can you provide the build output which fails?

**Comment by quentinalt-dintec** [comment]
ok so basically, I started all from the beginning and I went to look in the bin/gcc-native as you told me to.
I found the executable file of the quick_start_central_system and it looks like it is working
but then I cannot get my charger to connect to it

**Comment by quentinalt-dintec** [comment]
the examples are working well, when I run them in differents terminals, they connect to each other as expected.
What do I need to do/change to make my charger to connect to the quick_start_central_system ?
@c-jimenez

**Comment by c-jimenez** [comment]
Hi,

The quick_start_central_system example is configured to accept only TLS websocket connections.

To configure the example, edit the quick_start_central_system.ini file in the same directory :

- ListenUrl : URL to set in your charging station (use ws for unsecure HTTP and wss for HTTPS)
- HttpBasicAuthent : Set to false if not using OCPP security profile 1 or 2
- TlsClientCertificateAuthent: Set to false if not using OCPP security profile 3
